packages/react-ai/src/components/AIConversation/views/Controls/PromptControl.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { withBaseElementProps } from '@aws-amplify/ui-react-core/elements';
+
+import {
+  AIConversationElements,
+  InputContext,
+  SuggestedPromptsContext,
+} from '../../context';
+import { classNames } from '@aws-amplify/ui';
+
+const { View, Button, Text, Heading, Icon } = AIConversationElements;
+
+const PROMPT_BLOCK = 'ai-prompts';
+const PROMPT_CONTROL = `${PROMPT_BLOCK}__prompt`;
+const PROMPT_CARD = `${PROMPT_CONTROL}__card`;
+
+const PromptCardBase = withBaseElementProps(Button, {
+  className: PROMPT_CARD,
+});
+
+const PromptCard: typeof PromptCardBase = React.forwardRef(
```

Comment:
Just for my knowledge, are you defining `PromptCard` this way to add additional logic later on?

Reply:
Hm actually no. Good call out I think there isn't any additional logic we really need here so I can probably just use the card base